reporter.js

```diff
@@ -10,84 +10,90 @@ var wrStream;
 var outputFile;
 
 function encode(s) {
-  var pairs = {
-    "&": "&amp;",
-    '"': "&quot;",
-    "'": "&apos;",
-    "<": "&lt;",
-    ">": "&gt;"
-  };
-  for (var r in pairs) {
-    if (typeof(s) !== "undefined") {
-      s = s.replace(new RegExp(r, "g"), pairs[r]);
-    }
-  }
-  return s || "";
+	var pairs = {
```

Comment:
Sorry - I tried to match your indentation but it looks like my editor took over. Do you prefer tabs or spaces?